lib/LocalTopicModel.js

```diff
@@ -0,0 +1,464 @@
+/**
+ * Copyright 2016 BigML
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain
+ * a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+"use strict";
+var NODEJS = ((typeof module !== 'undefined') && module.exports);
+var PATH = (NODEJS) ? "./" : "";
+
+var ZERO_CODE = 48;
+var NINE_CODE = 57;
+var UPPER_BEGIN = 65;
+var UPPER_END = 90;
+var LOWER_BEGIN = 97;
+var LOWER_END = 122;
+var ASCII_END = 127;
+
+var MAXIMUM_TERM_LENGTH = 30;
+var UPDATES = 64;
+
+var CODE_TO_NAME = {
+    "da": "danish",
+    "nl": "dutch",
+    "en": "english",
+    "fi": "finnish",
+    "fr": "french",
+    "de": "german",
+    "hu": "hungarian",
+    "it": "italian",
+    "nn": "norwegian",
+    "pt": "portuguese",
+    "ro": "romanian",
+    "ru": "russian",
+    "es": "spanish",
+    "sv": "swedish",
+    "tr": "turkish"
+}
+
+if (NODEJS) {
+  var util = require('util');
+  var events = require('events');
+  var TopicModel = require(PATH + 'TopicModel');
+  var Snowball = require('snowball');
```

Comment:
the modules included here are only required if executed from node.js. The javascript version will not use them and they are usually related to functions not used in the local Topic Model code, but to the retrieval of the JSON resource. Are the snowball and mersenne-twister libraries correctly here?